CONTRIBUTING.md

```diff
@@ -1 +1 @@
-Please see: https://github.com/mattermost/platform/blob/master/CONTRIBUTING.md
+Please see: https://github.com/mattermost/mattermost-server/blob/master/CONTRIBUTING.md
```

Comment:
What you you think about just copying the text for the server repo?
```
# Code Contribution Guidelines

Thank you for your interest in contributing! Please see the [Mattermost Contribution Guide](https://developers.mattermost.com/contribute/getting-started/) which describes the process for making code contributions across Mattermost projects and [join our "Contributors" community channel](https://community.mattermost.com/core/channels/tickets) to ask questions from community members and the Mattermost core team.

When you submit a pull request, it goes through a [code review process outlined here](https://developers.mattermost.com/contribute/getting-started/code-review/).
```

Reply:
if the server repo change, we will need to change here as well. will keep as is for now